plugins/vpc-shared-eni/network/bridge_windows.go

```diff
@@ -263,6 +267,16 @@ func (nb *BridgeBuilder) FindOrCreateEndpoint(nw *Network, ep *Endpoint) error {
 	// Return network interface MAC address.
 	ep.MACAddress, _ = net.ParseMAC(hnsResponse.MacAddress)
 
+	// For task-networking, we need to remove the stale route entries from the routing table of default host compartment
+	if nw.TaskENIConfig.PauseContainer {
+		mask := net.ParseIP(staticRouteEntryMask)
+		err := routeutils.New().DeleteRoute(&ep.IPAddress.IP, &mask, nil)
```

Comment:
What happens if multiple attempts are made to delete the same route?
Would it be better to delete only if the required route is present?

Reply:
The route entry will be deleted only if it is present in the routing table of the host compartment. Otherwise nothing happens.
Error is returned only when the route command itself is not able to execute.